{"Modified":"2021-12-13T19:24:00","Published":"2021-12-08T12:15:00","access":{"authentication":"NONE","complexity":"LOW","vector":"NETWORK"},"assigner":"cve@mitre.org","capec":[{"id":"221","name":"XML External Entities Blowup","prerequisites":"A server that has an implementation that accepts entities containing URI values.","related_weakness":["611"],"solutions":"This attack may be mitigated by tweaking the XML parser to not resolve external entities. If external entities are needed, then implement a custom XmlResolver that has a request timeout, data retrieval limit, and restrict resources it can retrieve locally.","summary":"This attack takes advantage of the entity replacement property of XML where the value of the replacement is a URI. A well-crafted XML document could have the entity refer to a URI that consumes a large amount of resources to create a denial of service condition. This can cause the system to either freeze, crash, or execute arbitrary code depending on the URI."}],"cvss":6.4,"cvss-time":"2021-12-13T19:24:00","cvss-vector":"AV:N/AC:L/Au:N/C:P/I:N/A:P","cwe":"CWE-611","id":"CVE-2021-44557","impact":{"availability":"PARTIAL","confidentiality":"PARTIAL","integrity":"NONE"},"last-modified":"2021-12-13T19:24:00","references":["https://github.com/KBNLresearch/multiNER","https://github.com/KBNLresearch/multiNER/pull/3"],"summary":"National Library of the Netherlands multiNER <= c0440948057afc6e3d6b4903a7c05e666b94a3bc is affected by an XML External Entity (XXE) vulnerability in multiNER/ner.py. Since XML parsing resolves external entities, a malicious XML stream could leak internal files and/or cause a DoS.","vulnerable_configuration":[{"id":"cpe:2.3:a:kb:multiner:*:*:*:*:*:*:*:*","title":"cpe:2.3:a:kb:multiner:*:*:*:*:*:*:*:*"}],"vulnerable_configuration_cpe_2_2":[],"vulnerable_product":["cpe:2.3:a:kb:multiner:*:*:*:*:*:*:*:*"]}